lib/list.js

```diff
@@ -0,0 +1,17 @@
+const wreck = require('wreck');
+const Boom = require('boom');
+
+module.exports = async function(email) {
+  const config = this;
+
+  if (email === 'undefined') {
```

Comment:
If no params, then should pass back the full list - not a `401` - the list still exists.